### DISCUSSION TITLE
how to open db that was opened in another process, using sqliteStudio?

### DISCUSSION THREAD

**Discussion by garyLiuxh** [pawelsalawa/sqlitestudio#5262] [General]
### Details
 as above.

**Comment by pawelsalawa** [comment]
Normally. You just do so.

Remember, that the other process has to commit and pending transactions, that acquired locks on the database - during SQLiteStudio attempt to connect.

**Comment by garyLiuxh** [comment]
how to get a lock when open a dbFile using sqlIteStudio.

sample:
  process A open dbFile in python: conn = sqlite3.connect('dbFile?mode=rwc&uri=true&journal_mode=WAL&cache=shared', isolation_level=None)
 Process B open the dbFile in SQLiteStudio: how to do? 

thanks.

**Comment by pawelsalawa** [comment]
You add database to list of registered databases using "Add database" button in top left corner (or from Database menu). Then you fill in the database dialog. Once you accept it, the database will appear on the databases list in the left-side panel. Double click it and there it is - you're connected.

**Comment by Sierron** [comment]
Should probably converted into a discussion, no?
